diff -urN GG/src/Ogre/CMakeLists.txt GG-patched/src/Ogre/CMakeLists.txt
--- GG/src/Ogre/CMakeLists.txt	2013-05-10 16:42:54.000000000 +1100
+++ GG-patched/src/Ogre/CMakeLists.txt	2013-08-10 00:52:57.054551583 +1100
@@ -31,7 +31,7 @@
         )
     endif ()
 
-    target_link_libraries(GiGiOgre ${GiGiOgre_LINK_LIBS})
+    target_link_libraries(GiGiOgre ${GiGiOgre_LINK_LIBS} -pthread)
 
     # Installation of this library variant
     if (((NOT BUILD_SHARED_LIBS AND NOT RUNTIME_ONLY_PACKAGE) OR
