--- a/CMakeLists.txt.orig	2012-06-01 22:48:09.000000000 -0600
+++ b/CMakeLists.txt	2012-06-02 00:12:30.394251529 -0600
@@ -57,6 +57,9 @@ option(BUILD_OGRE_OIS_PLUGIN
 option(BUILD_DOCUMENTATION
        "Builds HTML documentation (requires Doxygen)."
        ON)
+option(BUILD_TESTS
+       "Builds test programs."
+       ON)
 
 set(RUNTIME_ONLY_PACKAGE false)
 set(DEVEL_ONLY_PACKAGE false)
@@ -302,7 +305,9 @@ endif ()
 ########################################
 add_subdirectory(src)
 
-add_subdirectory(test)
+if (BUILD_TESTS)
+    add_subdirectory(test)
+endif ()
 
 if (BUILD_TUTORIALS)
     add_subdirectory(tutorial)
